lib/objectid.js

```diff
@@ -0,0 +1,12 @@
+var ObjectId;
+
+try {
+  ObjectId = require('mongodb').ObjectId;
+} catch(e) {
+  if (e.code === 'MODULE_NOT_FOUND')
+    throw e
+}
+
```

Comment:
TBH, I don't like it either, but it was the simplest way to get the full power of an ObjectId (all it's validations, unique string generation, other functions etc) in one shot. The other option is to just not use mongodb at all and come up with our own unique implementation for each function. However, this is not a 5 point job. I figure in the long term proposal task I have for this sprint, we can decide whether we want to come up with our own implementations for each function or use source code from other projects, etc.